packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -3622,3 +3622,37 @@ mixin TextSelectionHandleControls on TextSelectionControls {
   @override
   void handleSelectAll(TextSelectionDelegate delegate) {}
 }
+
+/// [FloatingCursorSingleLongPressDetectorBuilder] is a mixin that adds handling for single long press on top of [TextSelectionGestureDetectorBuilder].
+/// This mixin is primarily used to handle floating cursor behavior during text selection.
+mixin FloatingCursorSingleLongPressDetectorBuilder on TextSelectionGestureDetectorBuilder{
```

Comment:
can this be incorporated into the base class instead of a separate mixin?

Reply:
Yes, I found that it doesn't affect `SelectableText`, so we can incorporate into the base class.